redash/query_runner/mssql_odbc.py

```diff
@@ -48,7 +48,7 @@ def configuration_schema(cls):
                 "verify_ssl": {
                     "type": "boolean",
                     "title": "Verify SSL certificate",
-                    "default": True,
+                    "default": False,
```

Comment:
Changing defaults can be a problem if we're not careful.

I *think* it should be ok in this instance though. :smile:

Reply:
there's a problem in ui, which doesn't set toggle to enabled state by default, when default True value is set in a schema

Reply:
k, that's something we should probably look at later on then. :smile: